SIL.WritingSystems.Tests/SldrTests.cs

```diff
@@ -31,6 +31,20 @@ public TestEnvironment(bool sldrOffline = true, DateTime? embeddedAllTagsTime =
 
 			private TemporaryFolder FolderContainingLdml { get; }
 
+			/// <summary>
+			/// Tries to get the LDML file for the requested ietfLanguageTag from the SLDR.
+			/// If unable to connect to SLDR, ignores the remainder of the test.
+			/// </summary>
+			/// <returns>The name the downloaded LDML file (to get the full path, combine with <see cref="FilePath"/></returns>
+			public string TryGetLdmlFile(string ietfLanguageTag)
```

Comment:
I wonder if it would be better to modify the existing `GetLdmlFile` method and add the `Assert.Ignore` there? It seems that the other places you modified where `GetLdmlFile` gets called could also use this check. But I didn't check if it's used elsewhere where this Assert doesn't make sense.